pkg/e2e/operators/osd_metrics_exporter.go

```diff
@@ -0,0 +1,79 @@
+package operators
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/config"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/spf13/viper"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	osdMetricsExporterTestPrefix   = "[Suite: operators] [OSD] OSD Metrics Exporter"
+	osdMetricsExporterBasicTest    = osdMetricsExporterTestPrefix + " Basic Test"
+	osdMetricsExporterEndpointTest = osdMetricsExporterTestPrefix + " Endpoint Test"
+)
+
+func init() {
+	alert.RegisterGinkgoAlert(osdMetricsExporterBasicTest, "SD_SREP", "Arjun Naik", "sd-cicd-alerts", "sd-cicd@redhat.com", 4)
+}
+
+var _ = ginkgo.Describe(osdMetricsExporterBasicTest, func() {
+	var (
+		operatorNamespace = "openshift-osd-metrics"
+		operatorName      = "osd-metrics-exporter"
+		clusterRoles      = []string{
+			"osd-metrics-exporter",
+		}
+		clusterRoleBindings = []string{
+			"osd-metrics-exporter",
+		}
+	)
+	h := helper.New()
+	checkClusterServiceVersion(h, operatorNamespace, operatorName)
+	checkDeployment(h, operatorNamespace, operatorName, 1)
+	checkClusterRoles(h, clusterRoles, true)
+	checkClusterRoleBindings(h, clusterRoleBindings, true)
+	checkUpgrade(helper.New(), operatorNamespace, operatorName, operatorName, "osd-metrics-exporter-registry")
+})
+
+var _ = ginkgo.FDescribe(osdMetricsExporterEndpointTest, func() {
```

Comment:
```suggestion
var _ = ginkgo.Describe(osdMetricsExporterEndpointTest, func() {
```
Should this be focused? I can't think why, but I'm happy to be educated :D

Reply:
It shouldn't be. It was left over after debugging.